setup.py

```diff
@@ -27,14 +27,14 @@ def run(self):
         )
         from django.core.management import call_command
         import django
-        if django.VERSION[:2] >= (1, 7):
+        if django.VERSION[:2] >= (1, 8):
```

Comment:
This guard is only for calling django.setup() .. which is required in 1.7 as well as 1.8.